definitions/managed sevices.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Managed sevices
+status: Feedback Appreciated
+category: property
+---
+## Managed sevices
+### What it is
+ A complete safety net to your technical environment. It is the trust that you can have in a team to look after, both proactively and reactively, an ever-changing database environment. Above all, it is the knowledge that you have a team of experts to cover all areas of issues!
```

Comment:
A little too marketing-y. This is a **really** good start but we'll need to be more succinct in our definition and a little less effusive with our praise of the tech. Ex: 

What is a managed service?  it is a service provided by a third party that can be consumed by application developers or operations teams. Managed services range from data storage, to monitoring, to analytics, or even text messages.

Reply:
@JasonMorgan I would gladly do the changes you are asking for. The next version would be comprising

- compressed paragraph, consistent with current glossary styling
- Won't sound like a Sales-representative(Hopefully)